build/webmethods_traceability_agent.yml

```diff
@@ -60,7 +63,7 @@ webmethods_traceability_agent:
     healthCheckInterval: ${STATUS_HEALTHCHECKINTERVAL:30s}
   log:
     level: ${LOG_LEVEL:info}
-    format: ${LOG_FORMAT:json}
+    format: ${LOG_FORMAT:line}
```

Comment:
Any reason, not to use json as default ?

Reply:
Nothing specific, i added it for better readability.